docs/docsite/rst/community/steering/community_topics_workflow.rst

```diff
@@ -4,121 +4,124 @@
    For other changes, create a `community topic <https://forum.ansible.com/new-topic?category=project&tags=community-wg>`_ to discuss them.
    (Creating a draft PR for this file and mentioning it in the community topic is also OK.)
 
-Ansible community topics workflow
-=================================
+.. _community_topics_workflow:
+
+Community topics workflow
+=========================
 
 Overview
 --------
 
-This document describes the Ansible community topics workflow (herein after ``Workflow``) to provide guidance on successful resolving topics in an asynchronous way.
+This document describes the Ansible Community Topics workflow (hereinafter Workflow) to provide guidance on successful resolving topics in the asynchronous way.
 
 The Workflow is a set of actions that need to be done successively within the corresponding time frames.
 
 .. note::
 
-   If you have any ideas on how the Workflow can be improved, please create an issue in this repository or pull request against this document.
+  This is a rough scenario and it can vary depending on a topic's complexity and other nuances, for example, when there is a mass agreement up-front.
 
 Creating a topic
 ----------------
 
-Any person can `create a topic <https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=project&tags=community-wg>`_ tagged with ``community-wg`` under the ``Project Discussions`` category in the `Ansible Forum <https://forum.ansible.com/>`_. A :ref:`Steering Committee member<steering_members>` can tag the forum post with `community-wg-nextmtg` to put it on the meeting agenda.
-
-Workflow
---------
-
-.. note::
-
-  This is a rough scenario and it can vary depending on a topic's complexity and other nuances, for example, when there is a mass agreement upfront.
+Any person can `create a topic <https://forum.ansible.com/new-topic?title=topic%20title&body=topic%20body&category=project&tags=community-wg>`_ tagged with ``community-wg`` under the ``Project Discussions`` category in the `Ansible Forum <https://forum.ansible.com/>`_.
```

Comment:
Given that the link already has the correct tag, category etc, can we just remove everything else in this sentence?

Reply:
Deliberately left it as people can be also interested in 1. just taking a look at the topics 2. subscribe to the topics, etc., so not sure

Reply:
After taking another look, i think that info is extra to have it here. Replaced the content with a link to another doc.